client/pages/docs/quote/_meta.json

```diff
@@ -1,8 +1,58 @@
 {
-	"index": "The `Quote` object",
-	"random": "Get random quote",
-	"random-via-anime": "Get random quote via anime",
-	"random-via-character": "Get random quote via character",
-	"quotes-by-anime": "Get quotes by anime",
-	"quotes-by-character": "Get quotes by character"
-}
+	"index": {
+		"title": "The `Quote` object",
+		"theme": {
+			"footer": true,
+			"sidebar": true,
+			"toc": false
+		}
+	},
+	"random": {
+		"title": "Get random quote",
+		"theme": {
+			"footer": true,
+			"sidebar": true,
+			"toc": false
+		}
+	},
+	"random-via-anime": {
+		"title": "Get random quote via anime",
+		"theme": {
+			"footer": true,
+			"sidebar": true,
+			"toc": false
+		}
+	},
+	"random-via-character": {
+		"title": "Get random quote via character",
+		"theme": {
+			"footer": true,
+			"sidebar": true,
+			"toc": false
+		}
+	},
+	"quotes-by-anime": {
+		"title": "Get quotes by anime",
+		"theme": {
+			"footer": true,
+			"sidebar": true,
+			"toc": false
+		}
+	},
+	"quotes-by-character": {
+		"title": "Get quotes by character",
+		"theme": {
+			"footer": true,
+			"sidebar": true,
+			"toc": false
+		}
+	},
+	"pagination": {
+		"title": "Pagination",
+		"theme": {
+			"footer": true,
+			"sidebar": true,
+			"toc": false
+		}
+	}
+}
```

Comment:
Is there any way to disable these settings globally for all pages instead of manually doing it for each of them?

Reply:
If a config file is set in meta.json inside a folder, it applies to all the files (sub-folders) inside the folder.

The config is now set globally on docs folder.

Fixed.

Reply:
Got it. Makes sense.
Is it possible to remove this last part from the right bar?

![image](https://github.com/user-attachments/assets/682e3483-1a66-42d1-a8b7-b9c4b57bf94f)

Reply:
Resolved.

Reply:
Great.